tests/selenium/test_markdown_copy_paste.py

```diff
@@ -7,9 +8,7 @@
 rawDf = pd.read_csv(url)
 rawDf["Complaint ID"] = rawDf["Complaint ID"].map(lambda x: "**" + str(x) + "**")
 rawDf["Product"] = rawDf["Product"].map(lambda x: "[" + str(x) + "](plot.ly)")
-rawDf["Issue"] = rawDf["Issue"].map(
-    lambda x: "![" + str(x) + "](https://dash.plot.ly/assets/images/logo.png)"
-)
```

Comment:
Ha good call - really shouldn't be depending on dash docs to be up and never change where or what the logo is!